src/components/Header/styled.js

```diff
@@ -13,4 +13,5 @@ export const HeaderWrapper = styled(Wrapper)`
     display: flex;
     justify-content: space-between;
     align-items: center;
+    gap: ${MEDIUM};
 `;
```

Comment:
are these changes still necessary?

Reply:
since we moved favorites to user avatar menu, i'll remove it thanks for the tip